includes/transformer/class-comment.php

```diff
@@ -63,8 +63,9 @@ public function to_object() {
 		$comment = $this->wp_comment;
 		$object = new Base_Object();
 
-		$object->set_id( $this->get_id() );
+		$object->set_id( $this->get_id( $comment ) );
```

Comment:
why this? we already have the comment as object var. no need to pass it to `get_id()`?!?

Reply:
we need somethings more flexible, so that if a reply is to a local post we can pass in the parent_comment

Reply:
but then I would use a helper function to be a bit more consistent